mainwindow.cc

```diff
@@ -445,7 +445,6 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
   closeCurrentTabAction.setShortcutContext( Qt::WidgetWithChildrenShortcut );
   closeCurrentTabAction.setShortcut( QKeySequence( "Ctrl+W" ) );
   closeCurrentTabAction.setText( tr("Close current tab") );
-  closeCurrentTabAction.setIcon( QIcon(":/icons/closetab-hover.png") );
```

Comment:
can this line left for windows only.I think it looks better than default.

Reply:
I just forced push the change. I added windows specific CSS code to custom close buttons.